src/main/java/org/apache/commons/lang3/builder/ToStringStyle.java

```diff
@@ -2633,4 +2647,48 @@ private Object readResolve() {
         }
 
     }
+
+    // ----------------------------------------------------------------------------
+
+    /**
+     * <p>
+     * <code>ToStringStyle</code> that outputs with No Class Name in a Multi Line format.
+     * </p>
+     *
+     * <p>
+     * This is an inner class rather than using
+     * <code>StandardToStringStyle</code> to ensure its immutability.
+     * </p>
+     *
+     * @since 3.7
+     * @see <a href="http://json.org">json.org</a>
+     */
+    private static final class NoClassNameMultiLineToStringStyle extends ToStringStyle {
+
+        private static final long serialVersionUID = 1L;
+
+        /**
+         * <p>Constructor.</p>
+         * <p>
+         * <p>Use the static constant rather than instantiating.</p>
+         */
+        NoClassNameMultiLineToStringStyle() {
+            super();
+            this.setUseClassName(false);
+            this.setUseIdentityHashCode(false);
+            this.setContentStart(StringUtils.EMPTY);
+            this.setFieldSeparator(System.lineSeparator());
+            this.setFieldSeparatorAtStart(false);
+            this.setContentEnd(System.lineSeparator());
+        }
+
+        /**
+         * <p>Ensure <code>Singleton</code> after serialization.</p>
+         *
+         * @return the singleton
+         */
+        private Object readResolve() {
+            return ToStringStyle.NO_CLASS_NAME_MULTI_LINE_STYLE;
```

Comment:
Really nit-picking here, but other classes in this file are referencing it without the class name (as this is the same class...). So if possible, for consistency it could be the same, i.e. `return NO_CLASS_NAME_MULTI_LINE_STYLE;`.

Reply:
Updated the return type.